cla_backend/libs/eligibility_calculator/cfe_civil/savings.py

```diff
@@ -18,23 +19,20 @@ def has_savings_key(request_data):
 
 
 def translate_savings(savings_data):
-    liquid = [
-        _savings_value(savings_data, "bank_balance", "Savings"),
-        _savings_value(savings_data, "investment_balance", "Investment")
-    ]
-    non_liquid = [
-        _savings_value(savings_data, "asset_balance", "Valuable items worth over 500 pounds")
-    ]
-    liquid_capital = none_filter(liquid)
-    non_liquid_capital = none_filter(non_liquid)
-
-    # all fields have to be set
-    if len(liquid_capital) + len(non_liquid_capital) < len(liquid) + len(non_liquid):
-        return {}
-    else:
+    if has_all_attributes(savings_data, ["bank_balance", "investment_balance", "asset_balance"]):
```

Comment:
The risk of a bug remains - if cla_public renames any of these fields, then this function will silently return nothing. Why so important to return `{}`?

Reply:
It's the presence of CFE_SAVINGS_KEY - translators return {} to indicate success or failure. Maybe they should return something more clear to indicate this - it's a challenge with the income/deductions anyway as they both return the same CFE key